src/app.js

```diff
@@ -49,7 +49,15 @@ const logLabel = 'App.js';
  */
 const makeResponse = (id, jsonrpc, data, isErr) => {
   const responseObj = { id, jsonrpc };
-  responseObj.result = isErr ? { Error: data } : data;
+  if (isErr) {
```

Comment:
This can be written like this:

```
return isErr ? { ...responseObj, error: data } : { ...responseObj, result: data };
```

Reply:
Made the edits, however, error object still requires its own separate object. Github is not picking up the edits for some reason.